### ISSUE TITLE
Unable to download binary from site

### ISSUE THREAD

**Issue by TheFern2** [radicle-dev/radicle-upstream#1859]
Is there another way to download the binary, I don't see binaries on github releases.
https://radicle.xyz/downloads.html download button renders and xml error:

![image](https://user-images.githubusercontent.com/10265682/117088898-6e39c700-ad19-11eb-8ad0-9dcd638685fb.png)

Right clicking and saving as is not downloading the full file. I got 127bytes.
```
0775 .rwxrwxr-x  46M fernandob 21 Mar 20:59 MKV.AppImage
0764 .rwxrw-r--  11M fernandob  8 Sep  2020 nvim.appimage
0775 .rwxrwxr-x  127 fernandob  4 May 20:36 radicle-upstream-0.2.4.AppImage
0755 .rwxr-xr-x  66M fernandob  9 Oct  2019 Scrivener-1.9.0.1beta-x86_64_english_only.AppImage
0764 .rwxrw-r-- 148M fernandob  7 Aug  2020 Ultimaker_Cura-4.6.2.AppImage
0764 .rwxrw-r--  81M fernandob 15 Aug  2020 UnityHub.AppImage

```

**Comment by TheFern2** [comment]
It looks like 0.2.4 file is not on the server, I can wget 0.2.3:

```
~/AppImages ❯❯❯ wget https://releases.radicle.xyz/radicle-upstream-0.2.4.AppImage
--2021-05-04 20:49:35--  https://releases.radicle.xyz/radicle-upstream-0.2.4.AppImage
Resolving releases.radicle.xyz (releases.radicle.xyz)... 35.244.208.180
Connecting to releases.radicle.xyz (releases.radicle.xyz)|35.244.208.180|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-05-04 20:49:36 ERROR 404: Not Found.

~/AppImages ❯❯❯ wget https://releases.radicle.xyz/radicle-upstream-0.2.3.AppImage
--2021-05-04 20:49:57--  https://releases.radicle.xyz/radicle-upstream-0.2.3.AppImage
Resolving releases.radicle.xyz (releases.radicle.xyz)... 35.244.208.180
Connecting to releases.radicle.xyz (releases.radicle.xyz)|35.244.208.180|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 111810958 (107M) [binary/octet-stream]
Saving to: ‘radicle-upstream-0.2.3.AppImage’

radicle-upstream-0.2.3.AppIm 100%[============================================>] 106.63M  26.1MB/s    in 4.4s    

2021-05-04 20:50:02 (24.3 MB/s) - ‘radicle-upstream-0.2.3.AppImage’ saved [111810958/111810958]

```

**Comment by geigerzaehler** [comment]
Hey @TheFern2. Thanks for reporting. We inadvertently updated the website to point to the upcoming release. The website is fixed and v0.2.4 should be released today.
